app/controllers/catalog_controller.rb

```diff
@@ -277,6 +291,7 @@ def index
   end
 
   def lazy_nonproject_tag_facet
```

Comment:
similarly, wondering if these methods could be a single method taking a field argument.

Reply:
These are controller endpoints.